sidechain/consensus/aura/src/block_importer.rs

```diff
@@ -159,6 +179,61 @@ impl<
 	pub(crate) fn block_author_is_self(&self, block_author: &SignedSidechainBlock::Public) -> bool {
 		self.authority.public() == *block_author
 	}
+
+	fn create_finish_game_extrinsic(
```

Comment:
```suggestion
	fn check_for_game_updates_and_create_finish_game_extrinsic_if_finished(
```
According to what I learnt from recently reading Clean Code: If a function name contains an "and" and "if", it contains too many functionalities. Maybe you can split here? That'd be great anyway, because the "checking" part will not be necessary in the hybrid way any more, but the sending part will likely be reused.